src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,89 @@
 package mate.academy.lib;
 
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
+import mate.academy.service.ProductService;
+import mate.academy.service.impl.FileReaderServiceImpl;
+import mate.academy.service.impl.ProductParserImpl;
+import mate.academy.service.impl.ProductServiceImpl;
+
 public class Injector {
+    private static final Map<Class<?>, Class<?>> IMPLEMENTATIONS = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<Class<?>, Class<?>> implementations = new HashMap<>();
```

https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

Reply:
Yeah, sure.